index.html

```diff
@@ -41,60 +35,161 @@ <h2 class="primary_color">I'm a software developer</h2>
+      </div>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img src="images/Snapshoot Portfolio.png" alt="Tonic Website Snapshoot" />
+        </a>
+        <h3 class="tertiary_color">Tonic</h3>
+        <ul class="d-flex">
+          <li class="li_works tertiary_color">CANOPY</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img
+            src="images/Snapshoot Portfolio-1.png"
+            alt="Multi-Post Stories Gain+Glory Snapshoot"
+          />
+        </a>
+        <h3 class="tertiary_color">Multi-Post Stories Gain+Glory</h3>
+        <ul class="d-flex justify-content-center">
+          <li class="li_works tertiary_color">Ruby on Rails</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+          <li class="li_works tertiary_color">html</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img src="images/Snapshoot Portfolio-2.png" alt="Tonic Website Snapshoot" />
+        </a>
+        <h3 class="tertiary_color">Tonic</h3>
+        <ul class="d-flex justify-content-between">
+          <li class="li_works tertiary_color">CANOPY</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img
+            src="images/Snapshoot Portfolio-3.png"
+            alt="Multi-Post Stories Gain+Glory Snapshoot"
+          />
+        </a>
+        <h3 class="tertiary_color">Multi-Post Stories</h3>
+        <ul class="d-flex justify-content-center">
+          <li class="li_works tertiary_color">Ruby on Rails</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+          <li class="li_works tertiary_color">html</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+    </section>
+
+    <!-- About me Section -->
+
+    <section class="d-flex flex-column align-items-center">
+      <article class="d-flex flex-column align-items-center border about_me">
+        <h2 class="pt-1 secondary_color">About Me</h2>
+        <p class="pt-1 pb-1 p_about_me tertiary_color">
+          I can help you build a product, feature or website. Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <a class="btn1 mb-1" href="#">Get My Resume</a>
+      </article>
+
+      <ul>
```

Comment:
- [ ] Great job using nested unordered list for this section. Kindly style the section to match the figma design. You are expected to stick to colors, padding and margins as much as possible.
### Your design
![f2](https://user-images.githubusercontent.com/94551484/205329685-f394d79b-bb33-41d3-9e97-2991810d0c9a.png)


### Figma template

![2](https://user-images.githubusercontent.com/94551484/205329789-d6284044-562f-4c8b-8f6c-b4d9bfe0e568.png)

Reply:
:red_circle:  For this section, KIndly follow the Figma design requirements as requested by the previous CR.You are expected to stick to colors, padding and margins as much as possible.

Your Design
<img src="https://user-images.githubusercontent.com/94551484/205329685-f394d79b-bb33-41d3-9e97-2991810d0c9a.png">

Figma Design
<img src="https://user-images.githubusercontent.com/94551484/205329789-d6284044-562f-4c8b-8f6c-b4d9bfe0e568.png">